### PR TITLE
Update the content of Package.md and README.md file & Replace icon with high resolution image

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
$subject related to https://github.com/ballerina-platform/ballerina-standard-library/issues/2657

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Package.md [hunk 1]
```diff
@@ -1,18 +1,63 @@
...
+};
+
+jwt:Payload result = check jwt:validate(jwt, validatorConfig);
+```
+
+## Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina standard library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+## Useful Links
```
Comment (shafreenAnfar) [inline ballerina/Package.md:60]: Don't we need the link to docs and BBE

**Review by shafreenAnfar — APPROVED** [review]

File: ballerina/Package.md (hunk 1 above)
Comment (ldclakmal) [reply ballerina/Package.md:60]: We only have these in the `README.md`, but not in the `Package.md`. Right? If it is needed, I will add those to `Package.md` as well.

Comment (shafreenAnfar) [reply ballerina/Package.md:60]: Yeah, we don't need it. We can add those later as central evolves.

Comment (ldclakmal) [reply ballerina/Package.md:60]: Ack.

**Pull request merged**